client/service/src/lib.rs

```diff
@@ -127,6 +127,7 @@ pub struct Service<TBl, TCl, TSc, TNetStatus, TNet, TTxPool, TOc> {
 	keystore: sc_keystore::KeyStorePtr,
 	marker: PhantomData<TBl>,
 	prometheus_registry: Option<prometheus_endpoint::Registry>,
+	_base_path: Option<BasePath>,
```

Comment:
Please add a comment why we need this here.

Reply:
done